### PR TITLE
etcd3 integration test api

### PR DESCRIPTION
**Pull request by Moriadry**
## What is the purpose of the change

fix https://github.com/apache/incubator-dubbo/issues/3631

## Brief changelog

XXXXX

## Verifying this change

XXXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Make sure there is a [GITHUB_issue](https://github.com/apache/incubator-dubbo/issues) field for the change (usually before you start working on it). Trivial changes like typos do not require a GITHUB issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
- [ ] Format the pull request title like `[Dubbo-XXX] Fix UnknownException when host config not exist #XXX`. Each commit in the pull request should have a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/incubator-dubbo/tree/master/dubbo-test).
- [ ] Run `mvn clean install -DskipTests=false` & `mvn clean test-compile failsafe:integration-test` to make sure unit-test and integration-test pass.
- [ ] If this contribution is large, please follow the [Software Donation Guide](https://github.com/apache/incubator-dubbo/wiki/Software-donation-guide).

### CODE REVIEW

**Comment by Moriadry** [comment]
@zonghaishang  hi, could pls help check this, many thanks :)

File: dubbo-configcenter/dubbo-configcenter-etcd/src/test/java/org/apache/dubbo/configcenter/support/etcd/EtcdDynamicConfigurationTest.java [hunk 1]
```diff
@@ -19,17 +19,17 @@
...
 import org.apache.dubbo.common.Constants;
 import org.apache.dubbo.common.URL;
 import org.apache.dubbo.configcenter.ConfigChangeEvent;
 import org.apache.dubbo.configcenter.ConfigurationListener;
-import org.junit.jupiter.api.AfterAll;
-import org.junit.jupiter.api.Assertions;
-import org.junit.jupiter.api.BeforeAll;
-import org.junit.jupiter.api.Disabled;
-import org.junit.jupiter.api.Test;
+import org.junit.*;
```
Comment (ralf0131) [inline dubbo-configcenter/dubbo-configcenter-etcd/src/test/java/org/apache/dubbo/configcenter/support/etcd/EtcdDynamicConfigurationTest.java]: It is not suggest to use `import *`. I'd like to keep them.

File: dubbo-dependencies-bom/pom.xml [hunk 2]
```diff
@@ -478,6 +480,18 @@
                 <version>${activation_version}</version>
             </dependency>
 
+            <!-- for dubbo-configcenter-etcd -->
```
Comment (ralf0131) [inline dubbo-dependencies-bom/pom.xml]: It is better to put all the modules to the `Test lib`. This make is clear that these are just for testing purpose.

Comment (Moriadry) [reply dubbo-dependencies-bom/pom.xml]: sorry, so where it should be? I found embedded-redis for test also placed in this dependencies-bom xml, and found no where else to put this dependency.

Comment (Moriadry) [reply dubbo-dependencies-bom/pom.xml]: oh, I got your idea!

**Comment by Moriadry** [comment]
Sure, thank you for your help.

**Pull request merged**